packages/@aws-cdk/aws-ec2/lib/vpc.ts

```diff
@@ -302,6 +322,25 @@ abstract class VpcBase extends Resource implements IVpc {
     });
   }
 
+  public addPeeringConnection(id: string, options: VpcPeeringConnectionOptions): VpcPeeringConnection {
+    return new VpcPeeringConnection(this, id, {
+      vpc: this,
+      ...options
+    });
+  }
+
+  public addRoute(id: string, options: RouteOptions): IRoute[] {
+    const {subnets, ...props} = options;
+    const subnetSelection = this.selectSubnetObjects(subnets || {});
+    const routes: IRoute[] = [];
+    subnetSelection.forEach((subnet: ISubnet, index: number) => {
+        routes.push(new Route(this, `${id}${index}${this.hashCidr(options.destinationCidr || options.destinationCidrIpv6)}`, {
```

Comment:
Feels like this should be a method on `ISubnet`, no?

Reply:
hmm, how should I handle the ids?
maybe this would work?
```typescript
interface ISubnet {
    addRoute(id: string, options: RouteOptions): Route
}
```